### DISCUSSION TITLE
Can live photo display and caching be supported

### DISCUSSION THREAD

**Discussion by CoderWeiLee** [onevcat/Kingfisher#1593] [General]
I have a demand for dynamic wallpaper display. I will download MP4 from the server and process it into. JPG and. Mov. How can I display a large number of live photos and cache them? Now I only download. MP4 in batches and convert it to live photo display

**Comment by onevcat** [comment]
:( Kingfisher now has nothing to do with Live Photos or videos like MP4. So I guess it is not supported here yet.
